src/libs/ReportUtils.ts

```diff
@@ -1415,6 +1412,23 @@ function canCreateTaskInReport(report: OnyxEntry<Report>): boolean {
     return true;
 }
 
+/**
+ * For all intents and purposes a report that has no notificationPreference at all should be considered "hidden".
+ * We will remove the 'hidden' field entirely once the backend changes for https://github.com/Expensify/Expensify/issues/450891 are done.
+ */
+function isHiddenParticipant(notificationPreference: string | null | undefined): boolean;
+function isHiddenParticipant(report: OnyxEntry<Report>): boolean;
+function isHiddenParticipant(reportOrPreference: OnyxEntry<Report> | string | null | undefined): boolean {
+    if (typeof reportOrPreference === 'object' && reportOrPreference !== null) {
+        const notificationPreference = getReportNotificationPreference(reportOrPreference);
+        return isHiddenParticipant(notificationPreference);
+    }
+    if (reportOrPreference === undefined || reportOrPreference == null) {
```

Comment:
Should we also check for an empty string? Also maybe strict equation with `null`


```suggestion
    if (reportOrPreference === undefined || reportOrPreference === null || reportOrPreference === '') {
```